src/reduxTypes.js

```diff
@@ -215,6 +215,19 @@ export type PresenceState = {
   [email: string]: UserPresence,
 };
 
+/**
+ * Disjoint union (tagged by the `name` property) that represents the set of
+ * video chat providers supported by the mobile app. Each member of the union
+ * represents a distinct video chat provider and encapsulates the associated
+ * data for creating video call links for that provider. For example, the type
+ * for the "jitsi_meet" provider contains a field for the jitsiServerUrl
+ * returned by the Zulip server.
+ *
```

Comment:
A couple of bits of doc-comment style:
* Generally we start a doc comment with a short summary, just a line or so.

  The Dart project generally has excellent documentation, and they have a great style guide for it, almost all of which can apply just as well in JS (or really most any language.) [This item](https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary), and the next few after it, are on point here.

* For a type's doc comment, the summary focuses on a value of that type, rather than on the type definition. See examples elsewhere in this file; e.g. `MessagesState` is "A map with all messages we've stored locally, indexed by ID", because `MessagesState` is the type *of* values like that. (Subsequent paragraphs can focus on the type definition, in cases where that's helpful.)

Reply:
Thank you for those pointers! I've updated the doc comment to reflect those guidelines.